xskillscore/core/comparative.py

```diff
@@ -223,16 +231,25 @@ def mae_test(
         first forecast to be compared to the observations
     forecasts2 : xarray.Dataset or xarray.DataArray
         second forecast to be compared to the observations
-    observations : xarray.Dataset or xarray.DataArray or None
+    observations : xarray.Dataset or xarray.DataArray, optional
         observations to be compared to both forecasts. if None, assumes that arguments
         forecasts1 and forecasts2 are already MAEs. Defaults to None.
-    time_dim : str
+    metric : str, optional
+        Name of distance metric function to be used for computing the error between forecasts
+        and observation. It can be any of the xskillscore distance metric function excep for
+        ``mape``. Valid metrics are ``me``, ``rmse``, ``mse``, ``mae``, ``median_absolute_error``,
+        and ``smape``. Note that if metric is None, observations must also be None.
+        Defaults to None.
+    time_dim : str, optional
         time dimension of dimension over which to compute the temporal correlation.
         Defaults to ``'time'``.
-    dim : str or list of str
-        dimensions to apply MAE to. Cannot contain ``time_dim``. Defaults to [].
-    alpha : float
+    dim : str or list of str, optional
+        dimensions to apply metric function to. Cannot contain ``time_dim``. Defaults to [].
```

Comment:
please specify that it is not possible to pass dim=None to the metrics and that dim=None is converted to dim=[].

Reply:
Done.